bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/GenericContentAccessResource.java

```diff
@@ -16,22 +16,27 @@
 package org.commonjava.indy.core.bind.jaxrs;
 
 import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiImplicitParam;
 import io.swagger.annotations.ApiOperation;
 import io.swagger.annotations.ApiParam;
 import io.swagger.annotations.ApiResponse;
 import io.swagger.annotations.ApiResponses;
 import org.commonjava.indy.bind.jaxrs.IndyDeployment;
 import org.commonjava.indy.bind.jaxrs.util.REST;
+import org.commonjava.indy.model.core.DemoteRequest;
+import org.commonjava.indy.util.ApplicationContent;
 import org.commonjava.maven.galley.event.EventMetadata;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
 import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Consumes;
 import javax.ws.rs.DELETE;
 import javax.ws.rs.GET;
 import javax.ws.rs.HEAD;
+import javax.ws.rs.POST;
```

Comment:
It seems like you are adding some imports without using them.

Reply:
Sure, good catch, forget it when I removed the interface.